language/tools/move-cli/README.md

```diff
@@ -75,45 +96,50 @@ a directory structure that mimics the Move memory model
 
 ### Project structure
 
-Each Move CLI project with a given `name` should have the following structure to
-it:
-
-```
-name/
-├── src # Directory containing all Move source modules
-│   ├ ...
-│   └── Module.move
-└── scripts # Directory containing all Move scripts
-    ├ ...
-    └── script.move
-```
-
-Let's now create a Move project that we'll use for the code in this README and `cd` into it:
+Each sandbox command is run in the context of a Move package. So let's create a
+Move package that we'll use for the code in this README and `cd` into it:
 
 ```shell
-$ mkdir readme
+$ move package new readme
 $ cd readme
-$ mkdir src
-$ mkdir scripts
 ```
 
 ### Compiling and running scripts
 
-Let's first start out with a simple script that prints its `signer`:
+Let's first start out with a simple script that prints its [`signer`](https://diem.github.io/move/signer.html):
 
 ```rust
 script {
 use Std::Debug;
-fun main(account: signer) {
+fun debug_script(account: signer) {
     Debug::print(&account)
 }
 }
 ```
 
-Place this in a file named `debug_script.move` under `scripts` and try
+Place this in a file named `debug_script.move` under `sources`.
+
+Before we can run this however, we need to import the Move standard library
+nursery in order to have access to the `Debug` module and `Std` [named
+address](https://diem.github.io/move/address.html#named-addresses).
+You can specify dependencies locally, or using a git URL. Here, we will specify
+it using git, so add the following to the `Move.toml` file in the `readme`
```

Comment:
So before you read further, please know that I know how incredibly pedantic this is, but: I believe the maintainers of "Git" the source control system intend for it to be capitalized. In [Git's documentation](https://git-scm.com/about), they capitalize it (from the first sentence at that link: "The Git feature that really makes it stand apart..."). So, I believe the executable is `git`, but the tool is called "Git," and as a result I'd suggest:

```suggestion
You can specify dependencies locally, or using a Git URL. Here, we will specify
it using Git, so add the following to the `Move.toml` file in the `readme`
```